guide/build-config.md

```diff
@@ -234,6 +235,46 @@ You need to specify `group`. Redhat artifacts are then sorted from builds includ
 
 The `MILESTONE` - not implemented yet
 
+The `RESOLVE_ONLY` strategy takes the entry points of the artifacts/extensions of of the product which are supposed to be packaged in repository, resolves the artifact, resolves artifacts managed dependencies. It also takes care of the transitives and package them as well.
```

Comment:
Double "of of" in this.
Do you mean

> The `RESOLVE_ONLY` strategy takes, as entry points, a list of the artifacts/extensions of the product which are to be packaged in the Maven repository. It resolves those artifacts and resolves their managed dependencies. It also takes care of the transitives and packages them as well.

?

Reply:
fixed "of" @RotBolt can you check rest of Nick's comments?